Marsiple/App/Todo Form/TodoFormViewController.swift

```diff
@@ -9,27 +9,29 @@
 import SnapKit
 
 class TodoFormViewController: UIViewController {
-    // NOTE: id is set to -1 so we can detect if we're creating new or changing existing todo (to detect wich initializer is used)
-    private var todo = Todo(id: -1, title: "", completed: false, userId: 1)
+    var todoTextIsEdited: (() -> Void)?
+    var todoCompletedIsEdited: (() -> Void)?
```

Comment:
I think we should have `one callback closure` per high-level operation - `Creation`, `Edit`.

Something along these lines: 
```
var todoCreationHandler: ((Todo) -> Void)?
var todoEditHandler: ((Todo) -> Void)?
```
or
```
var todoHandler: ((Todo, Bool) -> Void)?  
```

where `(Todo, Bool)` represents the `Todo` that modified or created and the `Bool` represents `isEditing`.